src/core/analytics.js

```diff
@@ -342,6 +354,7 @@ class Analytics {
               // logger.debug(msg);
               this.errorReporting.leaveBreadcrumb(msg);
               intgInstance = new intMod[modName](intg.config, self, intg.destinationInfo);
+              this.validateConfig(intgInstance.getRequiredConfigs(), intg.config, intg.name);
```

Comment:
`getRequiredConfigs` is not an appropriate term. `getRequiredConfigFields` is probably what you meant to use.